### ISSUE TITLE
TreeTable Keyboard Selection with Arrow Keys (Up/Down)

### ISSUE THREAD

**Issue by m-vignesh** [primefaces/primereact#1478]
### On Pressing the arrow key, only the focus shifts and not the selection. There should be an option to enable selection of rows directly on navigating with arrow keys in TreeTable

**I'm submitting a ...**  
```
[x]  feature request
```

**Current behavior**
Selection happens on pressing Enter after arrow (up/down)

**Expected behavior**
Selection should happen on pressing Arrows

**Environment:**

* **React version:**
16.8

* **PrimeReact version:**
4.2

* **Browser:** [all]

 
* **Language:** [TypeScript]

**Comment by melloware** [comment]
This is not how the TReetable keyboard support will work.  Arrows are for navigation not for selection.  See PrimeVue for the keyboard guidelines: https://primefaces.org/primevue/treetable
